gaTable.yaml

```diff
@@ -734,8 +734,30 @@
     parameters, as measured by a USGS real-time data-collection station, exceed user-definable thresholds.
   projectContact: 'Joseph Kalfsbeek'
   projectContactEmail: 'kalfsbek@usgs.gov'
-  analyticsContact: 'Hans Vraga'
-  analyticsContactEmail: 'hvraga@usgs.gov'
+  analyticsContact: 'Lauren Privette'
+  analyticsContactEmail: 'lprivette@usgs.gov'
+
+  - longName: "Surface Water Flow Trends in the Nation's Streams and Rivers"
```

Comment:
@lprivette Can you keep the indents here consistent?

Reply:
should be good now.  Is there a formatter you use in your IDE for this project or file? I can add it to my fork and avoid future issues like this. I'm terrible at noticing spacing differences.

Reply:
Hmm, not in particular.  Rstudio seems to maintain the indents fairly well on its own.  Anything that handles yaml files should do.